packages/cli/src/commands/generate/index.ts

```diff
@@ -1,12 +1,15 @@
-import { generate } from '@synthql/introspect';
 import path from 'path';
+import { generate } from '@synthql/introspect';
+import { isValidSchemaDefOverrides } from '../../validators/isValidSchemaDefOverrides';
+import { SchemaDefOverrides } from '@synthql/queries';
```

Comment:
SchemaDefOverrides is a CLI-only construct. It should therefore be defined in the CLI package.

Reply:
That is not correct 😅... It is also used in the introspect package

Reply:
Yeah about that: we also need to refactor that. We shouldn't couple the  /introspect package to the CLI. I'm about to post a comment around that.

Reply:
See https://github.com/synthql/SynthQL/pull/29#discussion_r1686162734